.env.example

```
@@ -20,5 +20,6 @@ NEXT_PUBLIC_EXCHANGE_URL=
 NEXT_PUBLIC_KVM_RELEASE_DATE_TIMESTAMP_MS=
 NEXT_PUBLIC_SWAP_URL=
 NEXT_PUBLIC_ENABLE_CONTRACT_VALIDATION=false
+NEXT_PUBLIC_ENABLE_3RD_PARTY_VALIDATION=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ENABLE_3RD_PARTY_VALIDATION key should go before the NEXT_PUBLIC_ENABLE_ASSET_APPLY key

(UnorderedKey)